File: hibernate/Hibernate_HQL/src/main/java/org/example/model/Survey.java
package org.example.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import java.time.LocalDate;
import java.time.LocalTime;

/**
 * Created by shankar on 4/7/2023.
 */

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Survey {

    @Id
    private int surveyId;
    private String  surveyName;
    private String  surveyLocation;
    private LocalDate surveyDate;
    private LocalTime surveyTime;


}

File: nse/angular/App2/src/app/survey/survey.ts
export class Survey {
    surveyId: number;
    surveyName: string;
    surveyDate: Date;
    surveyLocation: string;

    constructor(surveyId: number,
        surveyName: string,
        surveyDate: Date,
        surveyLocation: string
    ) {
        this.surveyId = surveyId;
        this.surveyName = surveyName;
        this.surveyDate = surveyDate;
        this.surveyLocation = surveyLocation;
    }


}
